### ISSUE TITLE
Make sure Android back gesture is handled in all modals

### ISSUE THREAD

**Issue by floatplane** [NWACus/avy#640]
We had some feedback that the swipe-to-go-back gesture was not working to exit our observation filter modals - does the approach you're using here fix that as well? If so - let's capture that fix in an issue to update the other modals as well!

_Originally posted by @stevekuznetsov in https://github.com/NWACus/avy/pull/638#discussion_r1430448845_

**Comment by yuliadub** [comment]
@stevekuznetsov can you repro on your android? i'll try on mine tonight

**Comment by rossobrien** [comment]
Correct, I see this issue when filtering observations on my android device. I'm currently setting up my development environment, once that's done I can start working on a fix
